manifests/qualys_eus_reporting.pp

```diff
@@ -0,0 +1,61 @@
+# @summary Allow qualys to sudo to root and add wrapper for subscription-manager to lie about enabled repos.
+#   See SVC-12072 for details
+#
+# @param enabled
+#   Boolean to enable/disable wrapper for subscription-manager
+#
+# @param repos
+#   Array of hashes containing repos to add to the qualys_eus_reporting.sh script
+#
+# @example
+#   include profile_audit::qualys_eus_reporting
+class profile_audit::qualys_eus_reporting (
+  Boolean     $enabled,
+  Array[Hash] $repos,
+){
+
+  # Alias that gets added to roots .bashrc
+  $bash_alias = 'function subscription-manager { /root/qualys_eus_reporting.sh "$@"; } #qualys_EUS_fake'
+
+  if ($enabled) {
+    $ensure_parm = 'present'
+
+    exec { 'add_qualys_EUS_alias':
+      path    => '/bin:/usr/bin',
+      command => "sed -i \'\$a${bash_alias}\' /root/.bashrc",
```

Comment:
What is the ‘\$a’ n this?

Reply:
`$a` makes it append it to the end of the .bashrc